boot/LoadLinux.c

```diff
@@ -344,7 +344,7 @@ void startLinux(void* initrdStart, unsigned long initrdSize, const char* appendL
 	BootIdeSetTransferMode(1, 0x40 | nAta);
 
 	// orange, people seem to like that colour
```

Comment:
You might want to change this comment to reflect the code change.

Reply:
Thank you. Fixed!